pandas/core/generic.py

```diff
@@ -6387,47 +6387,6 @@ def fillna(
         else:
             return result.__finalize__(self, method="fillna")
 
-    @overload
-    def ffill(
-        self: FrameOrSeries,
-        axis: None | Axis = ...,
-        inplace: Literal[False] = ...,
-        limit: None | int = ...,
-        downcast=...,
-    ) -> FrameOrSeries:
```

Comment:
will put the overloads back in once the deprecation warning's out, then we won't need as many